packages/lesswrong/components/editor/CKCommentEditor.tsx

```diff
@@ -7,6 +7,7 @@ import { ckEditorUploadUrlSetting, ckEditorWebsocketUrlSetting } from '../../lib
 import { ckEditorUploadUrlOverrideSetting, ckEditorWebsocketUrlOverrideSetting, forumTypeSetting } from '../../lib/instanceSettings';
 import { defaultEditorPlaceholder } from '../../lib/editor/make_editable';
 import { mentionPluginConfiguration } from "../../lib/editor/mentionsConfig";
+import {cloudinaryConfig} from '../../lib/editor/cloudinaryConfig.ts'
```

Comment:
I am surprised that whatever our module & build configuration is, it allows importing both with and without file extensions... but for now I think I'd prefer to keep it consistent (without)

Reply:
will update, annoyingly my IDE has a bug that all auto-imports have an extension regardless of configuration =\